bigtop-packages/src/deb/trino/control

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Source: trino
+Section: misc
+Priority: extra
+Maintainer: Bigtop <dev@bigtop.apache.org>
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.4.5
+
+Package: trino
+Architecture: any
+Depends: adduser, bigtop-utils (>= 0.7)
+Description:
+ trino is an open source distributed SQL query engine for running
+ interactive analytic queries against data sources of all sizes ranging
+ from gigabytes to petabytes.
+
+Package: trino-server
+Architecture: all
+Depends: trino (= ${source:Version}), python (>= 2.6)
```

Comment:
Dependency on python is tricky since it is not provided in recent distro like Ubuntu 22.04. You would need fallback dependency here like:
https://github.com/apache/bigtop/blob/master/bigtop-packages/src/deb/spark/control#L48